### PR TITLE
Fix gh-girdle to work on new github

### PR DESCRIPTION
**Pull request by sirpengi**
Haven't tested with all the event types yet (have to wait for them to show up on my feed), so some may be borked. But this is likely in better shape than what's currently in master.

### CODE REVIEW

File: ext/src/gh.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                '</div>' +
+            '</div>' +
+        '</div>'
+    );
+
+    var icon_template = _.template('<span class="octicon <%- icon %>" style="margin-right: 5px;" title="<%- title %>"></span>');
+
+		function extract_repo($el) {
+			var name = $el.text();
+			if (name.indexOf('/') === -1) {
```
Comment (dforsyth) [inline ext/src/gh.js:31]: branch names can have a `/` so this needs to search end to start

**Comment by dforsyth** [comment]
this looks cool.  what do you think about putting the user icons for people who have created the events we're squashing on the left side of the event count, like
`XXX/YYY had N events by [icon] [icon]` ?

I guess that could end up a bit wide, so we'd need to have some kind of truncation.  It's up to you though, if you don't want to do it in this change that is fine.

File: ext/src/gh.js (hunk 1 above)
Comment (sirpengi) [inline ext/src/gh.js:31]: Branch names can have slashes but not repo names, and github seems to render any activity related to branches like so:
githubbranchwithslash

Thus, the current method of extracting the Username/RepoName works as expected.

**Comment by sirpengi** [comment]
I could handle the truncation and whatnot, the real problem is that not all events come with a user thumbnail (in my feed, forking and starring don't show any user icons).  Also it's impossible to reverse the gravatar without access to any user's email address.  I guess that makes a few options:

1: don't do it
2: do it later
3: show icons but only those with a provided icon
4: show icons and fallback to placeholder image for users with no icon

Options 3 & 4 can use a register to keep track of all seen user icons, so icon will only be missing (or replaced with placeholder) if it is nowhere else in the feed.

I'd prefer option 2, I'd like to get this in working state and push it out for the general public.

**Comment by sirpengi** [comment]
There's also the public api (http://developer.github.com/v3/search/#search-users) but that's rate limited to 5 requests per minute for unauthenticated use.  The other option is to fetch the userpage and scrap that? Overkill?

File: ext/src/gh.js (hunk 1 above)
Comment (dforsyth) [inline ext/src/gh.js:31]: ah whoops i didn't see the `.last()` when selecting the element that you call this on.  all good.

**Comment by dforsyth** [comment]
lets go with 2.  i also couldn't get this to work in firefox, but we can fix that later i guess (or it could just be some freebsd weirdness).

**Pull request merged**